cli/flox-activations/src/activations.rs

```diff
@@ -59,6 +181,19 @@ struct AttachedPid {
     expiration: Option<OffsetDateTime>,
 }
 
+impl AttachedPid {
+    fn is_running(&self) -> bool {
```

Comment:
followup: we should factor out `pid_is_running` from https://github.com/flox/flox/pull/2192/files#diff-5b6d3c2231f49daa03ce723ee2c3575316d3055e404b653acbae321ada08a191 once 2192 merges and use that everywhere